### PR TITLE
Upgraded pipelines

### PR DESCRIPTION
**Pull request by ikrivosheev**
This is first part of: https://github.com/DevinR528/cargo-sort/pull/78

- [x] Upgraded version for plugins
- [x] Replaces use `actions-rs` to `dtolnay/rust-toolchain`

### CODE REVIEW

File: Cargo.toml
Comment (jplatte) [inline Cargo.toml:1]: The changes to this file seem unrelated.

**Review by jplatte — APPROVED** [review]

**Comment by ikrivosheev** [comment]
@jplatte I fixed `cargo sort`

**Comment by jplatte** [comment]
I don't like that. No idea what's making `cargo sort` complain about three separate comment lines one after the other, but.. let's just remove that step from CI? `cargo sort` was really always meant for large projects with big `Cargo.toml` files, using it on itself is kind of nice as a way of dogfooding, but it wouldn't catch regressions since it installs from `crates.io`.

**Comment by ikrivosheev** [comment]
> I don't like that. No idea what's making `cargo sort` complain about three separate comment lines one after the other, but.. let's just remove that step from CI?
> 
> `cargo sort` was really always meant for large projects with big `Cargo.toml` files, using it on itself is kind of nice as a way of dogfooding, but it wouldn't catch regressions since it installs from `crates.io`.

Yes, I removed `cargo sort` from CI
